src/dev/impl/DevToys/ViewModels/Tools/EncodersDecoders/JwtDecoderEncoder/JwtDecoder.cs

```diff
@@ -0,0 +1,293 @@
+﻿#nullable enable
+
+using System;
+using System.Composition;
+using System.IdentityModel.Tokens.Jwt;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using DevToys.Helpers;
+using DevToys.Helpers.JsonYaml;
+using DevToys.Models;
+using DevToys.Models.JwtDecoderEncoder;
+using Microsoft.IdentityModel.Logging;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.UI.Xaml.Controls;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.OpenSsl;
+
+namespace DevToys.ViewModels.Tools.EncodersDecoders.JwtDecoderEncoder
+{
+    [Export(typeof(JwtDecoder))]
+    [Shared]
+    public class JwtDecoder
```

Comment:
There's a lot of complexity in this class. Would it make sense to document it a little bit to help people who are unfamiliar with JWT like me? 😁